File: pkg/gate/testdata/contract-absence-present.go
package contractfix

// contract-absence-present.go (TASK-001): a real Go fixture in which a FORBIDDEN
// symbol is PRESENT, so the pack's grep absence probe MATCHES and the gate
// inverts it to a blocking absence VIOLATION (CLM-008). The forbidden token
// "legacyProbeSymbol" appears here (even in a comment-adjacent identifier), the
// conservative text-presence direction grep enforces.
func legacyProbeSymbol() string { return "should have been deleted" }

File: pkg/gate/testdata/ts-proof-pack/fixtures/ts/contract-absence-present.ts
// contract-absence-present.ts (TASK-002): a real .ts fixture in which a FORBIDDEN
// symbol is PRESENT, so the TS absence grep rule MATCHES and the gate inverts it
// to an absence VIOLATION (CLM-024). The forbidden token "legacyTsHelper" appears.
export function legacyTsHelper(): string {
  return "should have been removed";
}
